scriptmodules/emulators/xm7.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="xm7"
+rp_module_desc="Fujitsu FM-7 series emulator"
+rp_module_help="ROM Extensions: .d77 .t77 .d88 .2d \n\nCopy your FM-7 games to to $romdir/xm7\n\nCopy bios files DICROM.ROM, EXTSUB.ROM, FBASIC30.ROM, INITIATE.ROM, KANJI1.ROM, KANJI2.ROM, SUBSYS_A.ROM, SUBSYS_B.ROM, SUBSYSCG.ROM, SUBSYS_C.ROM, fddseek.wav, relayoff.wav and relay_on.wav to $biosdir/xm7"
+rp_module_section="exp"
+rp_module_flags="dispmanx !mali"
+
+function depends_xm7() {
+    getDepends libjpeg-dev libsdl1.2-dev libtool libpng12-dev libfreetype6-dev gawk fonts-takao
+    wget http://stable.hypertriton.com/agar/agar-1.5.0.tar.gz
+    tar xvfz agar-1.5.0.tar.gz
+    cd agar-1.5.0
+    ./configure --prefix="$md_build"
+    make -j1 depend
+    make -j1 all
+    make -j1 install
+}
+
+function sources_xm7() {
+    rm -rf XM7-for-SDL
+    git clone https://github.com/nakatamaho/XM7-for-SDL.git
+}
+
+function build_xm7() {
+    cd "$md_build"/XM7-for-SDL
+    mkdir linux-sdl/build
+    cd linux-sdl/build
+    CXXFLAGS="-DSHAREDIR='\"${md_inst}/share/xm7\"'" cmake -DCMAKE_INSTALL_PREFIX:PATH="$md_inst" -DCMAKE_BUILD_TYPE=Release -DUSE_OPENCL=No -DUSE_OPENGL=No -DWITH_LIBAGAR_PREFIX="$md_build" -DWITH_AGAR_STATIC=yes ..
+    make
+}
+
+function install_xm7() {
+    cd XM7-for-SDL/linux-sdl/build
+    make install
+}
+
+function configure_xm7() {
+    mkRomDir "xm7"
+    moveConfigDir "$home/.xm7" "$md_conf_root/xm7"
+    mkUserDir "$biosdir/xm7"
+    local bios
+    for bios in DICROM.ROM EXTSUB.ROM FBASIC30.ROM INITIATE.ROM KANJI1.ROM KANJI2.ROM SUBSYS_A.ROM SUBSYS_B.ROM SUBSYSCG.ROM SUBSYS_C.ROM fddseek.wav relayoff.wav relay_on.wav; do
+        ln -sf "$biosdir/xm7/$bios" "$md_conf_root/xm7/$bios"
+    done
+    addEmulator 1 "$md_id" "xm7" "$md_inst/bin/xm7 %ROM%"
+    addSystem "xm7"
+}
```

Comment:
the system is fm7 - the emulator is xm7 - so the config location / rom location should reflect the system not the emulator.